runtime.js

```diff
@@ -460,10 +460,12 @@
 
       // Pre-initialize at least 20 temporary variables to enable hidden
       // class optimizations for simple generators.
```

Comment:
I had assumed that temp variable usage was not guaranteed to be in order. I.e. `t1` could exist and not `t0`, if this isn't the case then your version would be much faster than the original but potentially slightly slower than the flagged version.